index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family='Poppins'&family='Inter':wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>My mobile portfolio</title>
+  </head>
+  <body>
+    <main>
+      <nav class="portfolio-nav">
+        <h2 class="portfolio-nav--logo"><a href="#">My Logo</a></h2>
+        <ul class="portfolio-nav--nav-options">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+        <button class="portfolio-nav--menu">
+          <img src="images/Icon-Menu.png" alt="menu button" />
+        </button>
+      </nav>
+      <section class="headline">
+        <h1 class="headline--name">I’m Nahom Glad to see you!</h1>
+        <p class="headline--desc">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h3>LET'S CONNECT</h3>
+        <ul class="headline--social-media">
```

Comment:
According to the project requirements, we must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please kindly use the same font style that is used in Figma 👍 

Your design
![image](https://user-images.githubusercontent.com/78251927/212207980-b394b387-bd68-4825-8e1b-76adbddfb413.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/212208360-25aab77d-179b-4268-a16e-9368179abbb0.png)

Reply:
Thanks for taking the time to review my project,@abdulhamiid. but is the only adjustments I should make are the font family right?

Reply:
@zdnahom 
Yes, just only the font style